signer/src/lib.rs

```diff
@@ -334,18 +335,112 @@ pub fn verify_signature(
     Ok(verify(&blob_to_sign, &signature_rs, &public_key))
 }
 
+fn verify_bls_signature(
+    signature: &SignatureBLS,
+    cbor_buffer: &CborBuffer,
+) -> Result<bool, SignerError> {
+    // TODO: need a function to extract from public key from cbor buffer directly
```

Comment:
Let's create an issue for this?

Reply:
Yes. This is not a priority but would avoid the parsing before and converting to pubkey.